src/index.js

```diff
@@ -130,9 +132,10 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
           events = [events];
         }
         events.forEach(eventName => {
-          const handlerOptions = !this.props.preventDefault && ['touchstart', 'touchmove'].indexOf(eventName) !== -1
-            ? { passive: true }
-            : null;
+          const handlerOptions =
```

Comment:
as mentioned - it breaks phantomjs tests:
- should not call handleClickOutside if this.props.disableOnClickOutside() is called, until this.props.enableOnClickOutside() is called.
- should attach and deattach event listener on updates
- should handle disabling and enabling onClickOutside listener when having no DOM node

As u can see all of them involves deattaching the DOM listener. To make it work we would have to store the reference to the object used while using `addEventListener` to pass it to `removeEventListener` later - solely to support phantomjs' quirk as its working just fine in real browsers.